core/tests/unit/ldml/keyboards/k_103_vkey.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+@@keys: [K_A][K_U][K_Q][K_C][K_K][K_SLASH][K_Z][K_Q][K_L][K_T][K_W]
+@@expected: \u0071\u0075\u0061\u0063\u006b\u002f\u0077\u0061\u006c\u0074\u007a
+-->
+
+<!DOCTYPE keyboard SYSTEM "../../../../../resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.dtd">
+<keyboard locale="en" conformsTo="techpreview">
+  <info author="srl295" indicator="🙀" layout="azerty" normalization="NFC" />
+
+  <names>
+    <name value="VKeyTestKbd" />
+  </names>
+
+  <vkeys>
+    <vkey from="Q" to="A" />
+    <vkey from="W" to="Z" />
+    <vkey from="A" to="Q" />
+    <vkey from="Z" to="W" />
+  </vkeys>
+
+  <keys>
+    <import base="cldr" path="techpreview/keys-Zyyy-punctuation.xml"/>
+  </keys>
+
+  <layers form="hardware" hardware="us">
+    <layer id="base" modifier="none">
+      <row keys="grave 1 2 3 4 5 6 7 8 9 0 hyphen equal" />
+      <row keys="q w e r t y u i o p open-square close-square backslash" />
+      <row keys="a s d f g h j k l semi-colon apos" />
+      <row keys="z x c v b n m comma period slash" />
```

Comment:
This shows the layout as qwerty but the vkeys are remapped to azerty. Is this intentional? If so, we should add a comment.

Reply:
> `a z e r t y`…